sdks/go/pkg/beam/runners/prism/internal/engine/elementmanager.go

```diff
@@ -628,6 +655,11 @@ type Block struct {
 	Transform, Family string
 }
 
+// StaticTimerID represents the static user identifiers for a timer.
+type StaticTimerID struct {
+	Transform, TimerFamily string
```

Comment:
Transform is the direct ID of a single transform, it's immaterial whether it's in a stage or not. 

Stages (and their IDs) represent fused, topologically sorted transforms. Essentially, within Beam, everything has it's own ID, and we generally refer to things by ID, instead of by direct code (except for Schema Row types, which is why they're awkward).

The goal is to avoid putting SDK specific information in Prism, so referring to an SDK specific implementation isn't useful, unless you already know that implementation.

Reply:
Can we get code comments for these properties?

Reply:
As discussed offline, I renamed the Transform field to TransformID, which makes comments on both fields redundant (they'd be of the form "TransformID is the ID of the transform containing the timer" etc).

Just to re-iterate the other point I made offline, documentation is best served in appropriate contexts. In this case, this is a type in an internal package, not intended for arbitrary user use, so the documentation bar is lower. That context is safe enough to assume that whomever needs to make a change here knows what a Timer is and how they are identified.

Otherwise they should really look into the beam Protos directly to start. (or some kind of SDK Authoring guide...)